scripts/postinstall.sh

```diff
@@ -45,3 +45,8 @@ else
     mkdir -p "android/app/src/main/res/raw/"
     cp $SOUNDS/* "android/app/src/main/res/raw/"
 fi
+
+APNG4Android="node_modules/APNG4Android"
+if [ ! -z "$APNG4Android" ]; then
+    git clone -b "fix/MM-62375-oom-on-gif" --single-branch https://github.com/rahimrahman/APNG4Android.git "$APNG4Android";
+fi
```

Comment:
Can't do `npm i` since APNG4Android doesn't have package.json.

TODO: will replace with mattermost/APNG4Android (as soon as permission to write is granted)

Reply:
No you cannot clone a specific commit, but you can clone the branch and then do a checkout

Reply:
You could also use --depth=1 so that it only clones the latest one